### PR TITLE
Refs #19514 - Expose qpid params to user

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by chris1984** [comment]
Set the max at 5000 per this Jira:
https://issues.apache.org/jira/browse/QPID-7677

File: manifests/init.pp [hunk 1]
```diff
@@ -63,6 +67,8 @@
   String $oauth_secret = $::katello::params::oauth_secret,
 
   String $post_sync_token = $::katello::params::post_sync_token,
+  Integer[0, 5000] $qpid_session_unacked = $::katello::params::qpid_session_unacked,
+  Integer[0, 1000] $qpid_wcache_page_size = $::katello::params::qpid_session_unacked,
```
Comment (ehelms) [inline manifests/init.pp]: Right hand side is  copy/paste error

File: manifests/init.pp [hunk 2]
```diff
@@ -18,6 +18,10 @@
 #
 # $repo_export_dir::    Directory to create for repository exports
 #
+# $qpid_session_unacked:: Buffer if the broker has a large number of sessions and the memory overhead is a problem
+#
+# $qpid_wcache_page_size:: The size (in KB) of the pages in the write page cache
```
Comment (ehelms) [inline manifests/init.pp]: I think these should be advanced parameters.

File: manifests/params.pp [hunk 3]
```diff
@@ -35,6 +35,10 @@
   # Pulp max speed setting
   $pulp_max_speed = undef
 
+  # Qpid perf settings
+  $qpid_session_unacked = 10
```
Comment (ekohl) [inline manifests/params.pp]: The qpid in EPEL doesn't accept this parameter and failing to start up. I think we should revert this.

**Pull request merged**